### PR TITLE
Update marketplace.xml

### PR DESCRIPTION
**Pull request by meishenlieshou**
Visualizer 6.06 version upgrade

### CODE REVIEW

File: marketplace.xml [hunk 1]
```diff
@@ -13163,6 +13159,8 @@ For more information <a target='NO_BLANK' href='http://www.datafor.com.cn/'>plea
       <screenshot>https://github.com/datafor123/visualizer-free/raw/master/resource/case1.jpg</screenshot>
       <screenshot>https://github.com/datafor123/visualizer-free/raw/master/resource/case2.jpg</screenshot>
       <screenshot>https://github.com/datafor123/visualizer-free/raw/master/resource/case3.jpg</screenshot>
+      <screenshot>https://github.com/datafor123/visualizer-free/raw/master/resource/case4.jpg</screenshot>
+      <screenshot>https://github.com/datafor123/visualizer-free/raw/master/resource/case5.jpg</screenshot>
```
Comment (dcleao) [inline marketplace.xml:13163]: Unfortunately, only three screenshots are supported.

**Review by dcleao — CHANGES_REQUESTED** [review]

**Review by meishenlieshou — COMMENTED** [review]
Delete the last two screenshots

File: marketplace.xml (hunk 1 above)
Comment (meishenlieshou) [reply marketplace.xml]: Thanks for checking it out! We've made the changes and submitted it again.

**Comment by meishenlieshou** [comment]
Visualizer version6.06 upgrade, with three screenshots

**Pull request merged**